app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find_by id: params[:followed_id]
+    current_user.follow @user
+    respond_to do |format|
+      format.html { redirect_to @user }
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find_by id: params[:id].followed
```

Comment:
plz create method for `Relationship.find_by id: params[:id]`
and remember handle when the result is nil

Reply:
is this right? im not sure